tests/src/Unit/OgAccessEntityTestBase.php

```diff
@@ -44,9 +44,9 @@ public function setup() {
 
     $this->groupManager->isGroup($entity_type_id, $bundle)->willReturn(FALSE);
 
-    $entity_manager = $this->prophesize(EntityManagerInterface::class);
-    $entity_manager->getFieldDefinitions($entity_type_id, $bundle)->willReturn([$field_definition->reveal()]);
-    \Drupal::getContainer()->set('entity.manager', $entity_manager->reveal());
+    $entity_field_manager = $this->prophesize(EntityFieldManagerInterface::class);
+    $entity_field_manager->getFieldDefinitions($entity_type_id, $bundle)->willReturn([$field_definition->reveal()]);
+    \Drupal::getContainer()->set('entity.manager', $entity_field_manager->reveal());
```

Comment:
To fix the failures, try changing this line to

`\Drupal::getContainer()->set('entity_field.manager', $entity_field_manager->reveal());`